lib/sanbashi.js

```diff
@@ -80,8 +80,12 @@ Sanbashi.filterByProcessType = function (jobs, procs) {
   return filteredJobs
 }
 
-Sanbashi.buildImage = function (dockerfile, resource, buildArg) {
+Sanbashi.buildImage = function (dockerfile, resource, buildArg, path) {
   let cwd = Path.dirname(dockerfile)
+  if (path) {
+    cwd = Path.join(process.cwd(), path)
```

Comment:
Sanbashi shouldn't use `process.cwd()`. If we need an absolute path here, it should be passed upstream.
Also, what if I do pass an absolute path to the command?

Reply:
I don't think an absolute path is needed here. I'll just pass whatever comes on --path as an arg to docker build